docs_user/modules/proc_migrating-existing-daemons-to-target-nodes.adoc

```diff
@@ -1,24 +1,20 @@
 [id="migrating-existing-daemons-to-target-nodes_{context}"]
 
-= Scenario 1: Migrating the existing daemons to the target nodes
+= Migrating the existing daemons to the target nodes
 
-Assuming we have 3 {CephCluster} nodes or ComputeHCI, this scenario extends the
-“monitoring” labels to all the {CephCluster} (or ComputeHCI) nodes that are part
-of the cluster. This means that we keep the count: 3 placements for the target
-nodes.
-//kgilliga: Note to self: This intro will be rewritten for GA. Revisit note in step 7 (placement and wording).
+The following procedure is an example of an environment with 3 {Ceph} nodes or ComputeHCI nodes. This scenario extends the monitoring labels to all the {Ceph} or ComputeHCI nodes that are part of the cluster. This means that you keep 3 placements for the target nodes.
 
 .Procedure
 
-. Add the monitoring label to all the {CephCluster} (or ComputeHCI) nodes in the cluster:
+. Add the monitoring label to all the {CephCluster} or ComputeHCI nodes in the cluster:
 +
 ----
 for item in $(sudo cephadm shell --  ceph orch host ls --format json | jq -r '.[].hostname'); do
     sudo cephadm shell -- ceph orch host label add  $item monitoring;
 done
 ----
 
-. Verify that all the (three) hosts have the monitoring label:
+. Verify that all 3 hosts have the monitoring label:
```

Comment:
Let's replace 3 with "target nodes", so we don't take strong assumptions on the Cluster size.